tests/common.py

```diff
@@ -55,7 +55,7 @@ def mongo():
 
 def mongo_cleanup(mongo):
     dbs = mongo.database_names()
-    dbs = [d for d in dbs if d not in ['local', 'admin']]
+    dbs = [d for d in dbs if d not in ['local', 'admin', 'config']]
```

Comment:
after upgrade to v3.6 do we need to change it for every service?

Reply:
yes

Reply:
I confirm this change is just fine and needs to be applied to a similar code across all the codebase.